lib/main.coffee

```diff
@@ -49,14 +38,18 @@ module.exports =
         filePath = textEditor.getPath()
         command = @executablePath
         ruleset = @rulesets
-        if @projectRules
-          rulesetPath = helpers.find(filePath, 'ruleset.xml')
+        if /^[a-z0-9]+\.xml$/gi.test(@rulesets)
+          rulesetPath = helpers.find(filePath, @rulesets)
           ruleset = rulesetPath if rulesetPath?
         parameters = []
         parameters.push(filePath)
         parameters.push('text')
         parameters.push(ruleset)
-        return helpers.exec(command, parameters).then (output) ->
+        options = {}
+        projectDir = atom.project.relativizePath(filePath)[0]
+        if projectDir?
+          options.cwd = projectDir
```

Comment:
Would it make any sense to do an `else` setting it to the directory of the file if it isn't open in an Atom project?